web/src/app/profile/approved-researcher/components/AgreementForm.css

```diff
@@ -1,32 +1,23 @@
-.form-container {
+.form__wrapper {
   max-width: 400px;
   margin: 2rem auto;
   padding: 1rem;
 }
 
-.input-checkbox {
+input[name="agreed"][type="checkbox"] {
   margin: 0.5rem;
 }
 
-.submit-button {
+.button--submit {
```

Comment:
I think `btn` is used elsewhere so we should aim to be consistent. I usually go for the longer form, but it's up to you.

Reply:
+1 for `button`